web-client/src/presenter/computeds/formattedWorkQueue.ts

```diff
@@ -272,9 +272,15 @@ export const getWorkItemDocumentLink = ({
   const baseDocumentLink = `/case-detail/${workItem.docketNumber}/documents/${workItem.docketEntry.docketEntryId}`;
   const documentViewLink = `/case-detail/${workItem.docketNumber}/document-view?docketEntryId=${workItem.docketEntry.docketEntryId}`;
 
+  const { USER_ROLES } = applicationContext.getConstants();
   let editLink = documentViewLink;
   if (showDocumentEditLink) {
-    if (permissions.DOCKET_ENTRY) {
+    if (
```

Comment:
It feels strange to me that the web-client needs to do so much computation to understand if a link is editable. It feels as if the function to determine if a link is editable should be based off of permissions and come from our entities. As in we should have a shared function between frontend and backend around if a document is editable.

Reply:
We think that what we're modifying here is view logic and not really business logic that makes sense to share between back/front -- we're not computing whether or not the given user can edit, only providing a 'shortcut' in the view to get to the edit screen. But we're open to further discussion.

Reply:
I see what you are saying. I thought the editability of this was being derived based off of permissions and that is incorrect.

Reply:
Types, coming right up!